test/with-dag-api.js

```diff
@@ -171,13 +173,15 @@ describe('with dag-api', () => {
         }
       }
 
-      before((done) => {
+      before(function (done) {
+        this.timeout(20 * 1000)
+
         node = new IPFS({
-          repo: '/tmp/unixfs-test-' + Math.random(),
+          repo: path.join(os.tmpdir(), 'unixfs-test-' + Math.random()),
           start: false
         })
 
-        node.on('ready', done)
```

Comment:
Does this not already log the error if there is one?

Reply:
Sorry, it was just debug code.  I'll revert to original.